app/components/track-form/component.js

```diff
@@ -48,30 +49,15 @@ export default Component.extend({
 		}
 	}),
 
-	automaticGetDiscogsInfo: observer('track.discogsUrl', async function () {
-		const track = get(this, 'track');
-
-		// Can not continue without a track or URL.
-		if (!track || !track.get('discogsUrl')) {
-			return;
-		}
-
-		// Because the URL might have changed
-		const mediaUrl = mediaUrlParser(track.get('discogsUrl'))
-		if (mediaUrl.id && mediaUrl.provider === 'discogs') {
-			const type = mediaUrl.url.includes('master/') ? 'master' : 'release'
-			const info = await fetchDiscogsInfo(mediaUrl.id, type)
-			this.set('discogsInfo', info)
+	automaticSetDiscogsInfo: observer('track.discogsUrl', function () {
```

Comment:
`automaticFetchDiscogsInfo`?

Reply:
yea also fine. tried to align with `automaticSetTitle`.